docs/history.rst

```diff
@@ -4,8 +4,8 @@ Changelog
 3.0.5 (unreleased)
 ------------------
 
-- Fix commit hook bug when content has been moved
-  [instification]
```

Comment:
@flavico please don't remove the previous entry. Add yours above the previous one.

Reply:
You haven't done that yet